### DISCUSSION TITLE
How to figure out source of code from "Code from unknown sourcefiles"

### DISCUSSION THREAD

**Discussion by mn-prp** [parcel-bundler/parcel#8217] [Q&A] [answered]
When running `parcel build src/index.html --detailed-report`, I get a huge portion of the build being listed as "code from unknown sourcefiles" (around 2MB each for two generated files). 

That's well over half of the generated code being from unknown source files. How can I figure out where this is coming from? If I understand correctly, this means that the source maps are not complete, but I don't know why.

**Comment by mischnic** [comment, marked as answer]
See https://github.com/parcel-bundler/parcel/issues/7924#issuecomment-1146163662

Load your file into https://evanw.github.io/source-map-visualization/ and look for unmapped sections

**Comment by ChristophP** [comment]
I'm also seeing a huge chunk of the bundle as "Code from unmapped sources"

<img width="679" alt="grafik" src="https://github.com/parcel-bundler/parcel/assets/5880236/7357d631-a375-459e-86a6-079cfe7ee625">

Just for my understanding: If the cause of this is broken source maps, does that mean that libraries are shipping broken source maps or is that something with source map generation done by parcel?

**Reply by mischnic** [reply]
Probably Parcel, can you share a reproduction?
